src/config.ts

```diff
@@ -18,10 +18,10 @@ const conf = {
     defaultDuration: 1 * UNIT_TIME.MINUTE,
     /** Options in interval duration */
     intervalDuration: {
-      60: 'Last minute',
-      300: 'Last 5 minutes',
-      600: 'Last 10 minutes',
-      1800: 'Last 30 minutes',
+      60: 'Last min',
+      300: 'Last 5 min',
+      600: 'Last 10 min',
+      1800: 'Last 30 min',
```

Comment:
@lucasponce @jotak @mwringe Are you ok with these changes?
This will change some other texts outside the graph toolbar.

The abbreviated units appear in the UXD screenshots: https://github.com/kiali/kiali-design/pull/45, but not sure if we want the abbreviations in the other places.

Reply:
+1 
Not strong opinion here.

Reply:
no problem